src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -501,12 +497,7 @@ def reconstruct_messages(
                     kafka_payload = KafkaPayload(
                         key=message.payload.key,
                         value=rapidjson.dumps(new_payload_value).encode(),
-                        headers=[
-                            *message.payload.headers,
-                            ("mapping_sources", mapping_header_content),
```

Comment:
I just realized, this is technically still used by the last seen updater, so I'm not sure we can safely delete these headers while that consumer is still around.

Reply:
We should be fine, as I think we'll just hit this condition in the last seen updater https://github.com/getsentry/sentry/blob/master/src/sentry/sentry_metrics/consumers/last_seen_updater.py#L52-L65 and the message will be ignored/dropped. 

Still, maybe let's make a note of this -- that by removing the headers the last seen updater will no longer be updating the last_seen column in PG.